programmers/3일차/K번째-수.js

```diff
@@ -0,0 +1,17 @@
+function solution(array, commands){
+    
+    const result = [];
+
+    for ( let eachCommand of commands ) {
+        let arrayEachCommand = [];
```

Comment:
```suggestion
         arrayEachCommand = [];
```
for 문에서 변수를 선언하는 것은  가급적이면 지양해야 합니다  반복문이 돌 때마다  변수가 선언되기 때문이에요
미리 위에서 선언 해준 다음  이런 식으로  변수를 초기화 해주시면 될 거 같아요